#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Update to latest CVS
# DP: Date: 2006-02-15

Index: locale/iso-4217.def
===================================================================
--- locale/iso-4217.def.orig
+++ locale/iso-4217.def
@@ -9,7 +9,7 @@
  * !!! The list has to be sorted !!!
  */
 DEFINE_INT_CURR("AED")		/* United Arab Emirates Dirham  */
-DEFINE_INT_CURR("AFA")		/* Afghanistan Afgani  */
+DEFINE_INT_CURR("AFN")		/* Afghanistan Afgani  */
 DEFINE_INT_CURR("ALL")		/* Albanian Lek  */
 DEFINE_INT_CURR("AMD")		/* Armenia Dram  */
 DEFINE_INT_CURR("ANG")		/* Netherlands Antilles  */
@@ -18,10 +18,10 @@
 DEFINE_INT_CURR("AUD")		/* Australian Dollar  */
 DEFINE_INT_CURR("AWG")		/* Aruba Guilder  */
 DEFINE_INT_CURR("AZM")		/* Azerbaijan Manat  */
-DEFINE_INT_CURR("BAK")		/* Bosnian and Herzegovina Convertible Mark  */
+DEFINE_INT_CURR("BAM")		/* Bosnian and Herzegovina Convertible Mark  */
 DEFINE_INT_CURR("BBD")		/* Barbados Dollar  */
 DEFINE_INT_CURR("BDT")		/* Bangladesh Taka  */
-DEFINE_INT_CURR("BGL")		/* Bulgarian Lev  */
+DEFINE_INT_CURR("BGN")		/* Bulgarian Lev  */
 DEFINE_INT_CURR("BHD")		/* Bahraini Dinar  */
 DEFINE_INT_CURR("BIF")		/* Burundi Franc  */
 DEFINE_INT_CURR("BMD")		/* Burmudian Dollar  */
@@ -76,6 +76,7 @@
 DEFINE_INT_CURR("IQD")		/* Iraqi Dinar  */
 DEFINE_INT_CURR("IRR")		/* Iranian Rial  */
 DEFINE_INT_CURR("ISK")		/* Iceland Krona  */
+DEFINE_INT_CURR("JEP")		/* Jersey Pound  */
 DEFINE_INT_CURR("JMD")		/* Jamaican Dollar  */
 DEFINE_INT_CURR("JOD")		/* Jordanian Dinar  */
 DEFINE_INT_CURR("JPY")		/* Japanese Yen  */
@@ -98,7 +99,7 @@
 DEFINE_INT_CURR("LYD")		/* Libyan Arab Jamahiriya Dinar  */
 DEFINE_INT_CURR("MAD")		/* Moroccan Dirham  */
 DEFINE_INT_CURR("MDL")		/* Moldova Lei  */
-DEFINE_INT_CURR("MGF")		/* Madagasy Franc  */
+DEFINE_INT_CURR("MGA")		/* Madagasy Ariary  */
 DEFINE_INT_CURR("MKD")		/* Macedonia Denar  */
 DEFINE_INT_CURR("MMK")		/* Myanmar Kyat  */
 DEFINE_INT_CURR("MNT")		/* Mongolia Tugrik  */
@@ -108,7 +109,7 @@
 DEFINE_INT_CURR("MUR")		/* Mauritius Rupee  */
 DEFINE_INT_CURR("MVR")		/* Maldives Rupee  */
 DEFINE_INT_CURR("MWK")		/* Malawi Kwacha  */
-DEFINE_INT_CURR("MXP")		/* Mexican Peso  */
+DEFINE_INT_CURR("MXN")		/* Mexican Peso  */
 DEFINE_INT_CURR("MYR")		/* Malaysian Ringgit  */
 DEFINE_INT_CURR("MZM")		/* Mozambique Metical  */
 DEFINE_INT_CURR("NAD")		/* Namibia Dollar  */
@@ -127,7 +128,8 @@
 DEFINE_INT_CURR("PYG")		/* Paraguay Guarani  */
 DEFINE_INT_CURR("QAR")		/* Qatar Rial  */
 DEFINE_INT_CURR("ROL")		/* Romanian Leu  */
-DEFINE_INT_CURR("RUR")		/* Russian Ruble  */
+DEFINE_INT_CURR("RON")		/* Romanian New Leu  */
+DEFINE_INT_CURR("RUB")		/* Russian Ruble  */
 DEFINE_INT_CURR("RWF")		/* Rwanda Franc  */
 DEFINE_INT_CURR("SAR")		/* Saudi Arabia Riyal  */
 DEFINE_INT_CURR("SBD")		/* Solomon Islands Dollar  */
@@ -147,11 +149,11 @@
 DEFINE_INT_CURR("SYP")		/* Syrian Arab Republic Pound  */
 DEFINE_INT_CURR("SZL")		/* Swaziland Lilangeni  */
 DEFINE_INT_CURR("THB")		/* Thai Baht  */
-DEFINE_INT_CURR("TJR")		/* Tajikistani Ruble  */
+DEFINE_INT_CURR("TJS")		/* Tajikistani Somoni  */
 DEFINE_INT_CURR("TMM")		/* Turkmenistan Manet  */
 DEFINE_INT_CURR("TND")		/* Tunisian Dinar  */
 DEFINE_INT_CURR("TOP")		/* Tonga Pa'Anga  */
-DEFINE_INT_CURR("TRL")		/* Turkish Lira  */
+DEFINE_INT_CURR("TRY")		/* New Turkish Lira  */
 DEFINE_INT_CURR("TTD")		/* Trinidad and Tobago  */
 DEFINE_INT_CURR("TVD")		/* Tuvalu Dollars  */
 DEFINE_INT_CURR("TWD")		/* Taiwan, Province of China Dollar  */
@@ -171,7 +173,6 @@
 DEFINE_INT_CURR("XOF")		/* West African Franc (Benin, Ivory Coast, Niger, Senegal, Togo, Upper Volta)  */
 DEFINE_INT_CURR("XPF")		/* French polynesia, New Caledonia, Wallis and Futuna Islands  */
 DEFINE_INT_CURR("YER")		/* Yemeni Rial  */
-DEFINE_INT_CURR("YUM")		/* Yugoslavian New Dinar  */
 DEFINE_INT_CURR("ZAR")		/* South Africa Rand (Lesotho, Namibia)  */
 DEFINE_INT_CURR("ZMK")		/* Zambian Kwacha  */
 DEFINE_INT_CURR("ZWD")		/* Zimbabwe Dollar  */
